templates/books/book_detail.html

```diff
@@ -5,36 +5,21 @@
 
   {% if user.is_authenticated %}
     {% if book.owner == user or user.is_superuser %}
+    <div class="uk-container-expand uk-margin-top">
+    {% url 'book_update' pk=book.id as book_update %}
+    {% url 'book_delete' pk=book.id as book_delete %}
+    <nav class="book-menu maxw600">
+      <ul class="uk-nav nav-bookupdate flex-nav-books">
+        <li><a class="uk-button uk-button-bookupdate
+        {% if request.get_full_path == book_update %} active{% endif %}"
+         href="{% url 'book_update' pk=book.id %}">Update this book</a></li>
```

Comment:
re-use the urls from above here and on delete below ? ;)

Reply:
okay :)